pkg/solana/chainwriter/transform_registry.go

```diff
@@ -32,19 +34,43 @@ func FindTransform(id string) (func(context.Context, any, solana.AccountMetaSlic
 
 // This Transform function looks up the token pool addresses in the accounts slice and augments the args
 // with the indexes of the token pool addresses in the accounts slice.
-func CCIPExecuteArgsTransform(ctx context.Context, args any, accounts solana.AccountMetaSlice, tableMap map[string]map[string][]*solana.AccountMeta) (any, solana.AccountMetaSlice, error) {
+func CCIPExecuteArgsTransform(ctx context.Context, args any, accounts solana.AccountMetaSlice, tableMap map[string]map[string][]*solana.AccountMeta) (any, solana.AccountMetaSlice, []txmutils.SetTxConfig, error) {
```

Comment:
nit: Since these methods have grown recently, maybe we should just call them `CCIPExecuteTransform` and `CCIPCommitTransform` now to keep it a little more generic